reference/tools/gnu/autotoolstoolchain.rst

```diff
@@ -53,6 +53,11 @@ corresponding build flags like ``-stdlib=libstdc++``, ``-std=gnu14``, architectu
 etc. It will also append the folder where the Conan generators are located to the
 ``PKG_CONFIG_PATH`` environment variable.
 
+Since `Conan 2.4.0 <https://github.com/conan-io/conan/releases/tag/2.4.0>`__,
+if it's a cross-building context, ``CC_FOR_BUILD`` and ``CXX_FOR_BUILD`` are also set as environment variables if the
+build profile is defining the ``c`` and ``cpp`` values in the conf variable ``tools.build:compiler_executables``.
+See more info in the :ref:`conf section<conan_tools_gnu_autotoolstoolchain_conf>`.
```

Comment:
```suggestion
Since `Conan 2.4.0 <https://github.com/conan-io/conan/releases/tag/2.4.0>`__,
in  a cross-building context, the environment variables ``CC_FOR_BUILD`` and ``CXX_FOR_BUILD`` are also set if the
build profile defines the ``c`` and ``cpp`` values in the configuration variable ``tools.build:compiler_executables``.
See more info in the :ref:`conf section<conan_tools_gnu_autotoolstoolchain_conf>`.
```